wamr-compiler/CMakeLists.txt

```diff
@@ -45,6 +45,7 @@ add_definitions(-DWASM_ENABLE_DUMP_CALL_STACK=1)
 add_definitions(-DWASM_ENABLE_PERF_PROFILING=1)
 add_definitions(-DWASM_ENABLE_LOAD_CUSTOM_SECTION=1)
 add_definitions(-DWASM_ENABLE_LIB_WASI_THREADS=1)
+add_definitions(-DWASM_CONFIGUABLE_BOUNDS_CHECKS=1)
```

Comment:
Seems no need to change here as no file is changed under `core/iwasm/compilation`.